js_modules/dagit/packages/core/src/assets/AssetGroupRoot.tsx

```diff
@@ -29,6 +30,8 @@ export const AssetGroupRoot: React.FC<{repoAddress: RepoAddress; tab: 'lineage'
   repoAddress,
   tab,
 }) => {
+  useTrackPageView();
```

Comment:
Yeah, I noted this in the diff summary. The idea is to track page views with the route itself, not the full location. E.g. aggregate tracking of `/instance/runs/:runID` page views instead of treating every individual `/instance/runs/{id}` page view as a different page. We can't really do that using a top-level `useLocation`, since we can't match the route anywhere except within the route.